internal/pkg/utils/stripe_utils.go

```diff
@@ -19,9 +19,7 @@ func setStripeKey(isTest bool) {
 	} else {
 		stripe.Key = stripeLiveKey
 	}
-	stripe.DefaultLeveledLogger = &stripe.LeveledLogger{
-		Level: 0,
-	}
+	stripe.DefaultLeveledLogger = &stripe.LeveledLogger{Level: 0}
```

Comment:
This should be equivalent. I would expect this to already be the definition of the default logger though?
```suggestion
	stripe.DefaultLeveledLogger = &stripe.LeveledLogger{}
```

Reply:
The default logger uses `Level: LevelError` which is `1`, we're setting the default to `LevelNull`.

Your suggested change should still be equivalent. `Level` is defined by `type Level int32`, and the default value of `int32` is `0`.

Think we're safe to make this change, or leave it for now?

Reply:
Let's leave as-is. It's better to be explicit in this case.